scaling.adoc

```diff
@@ -180,25 +180,25 @@ across multiple machines. While this is a prevalent scaling approach
 among popular websites and databases, it's not easily done in
 Bitcoin.
 
-Many people refer to this Bitcoin scaling approach as _sharding_. You
-let each full node verify just a part of the blockchain. Peter Todd
-has put a lot of thought into the concept of sharding. He
-https://petertodd.org/2015/why-scaling-bitcoin-with-sharding-is-very-hard[wrote
-a blog post] explaining sharding from a high level, and also presented
+Many people refer to this Bitcoin scaling approach as _sharding_. Basically, it consists in letting
+each full node verify just a portion of the blockchain. Peter Todd
+has put a lot of thought into the concept of sharding. He wrote a
+https://petertodd.org/2015/why-scaling-bitcoin-with-sharding-is-very-hard[blog
+post] explaining sharding from a high level [comm: do you mean to say that he explained it roughly? I'm not sure of what "from a high level" means], and also presenting
 his own idea called _treechains_. The article is a difficult read,
-but he makes some general points that are more digestible.
+but Todd makes some general points that are quite digestible.
```

Comment:
Maybe we should skip the word "general" here if we use "general terms" from my previous comment.